### PR TITLE
Update cs.po

### PR DESCRIPTION
**Pull request by groundf**
Replace "vyvádět" by "exportovat".

### CODE REVIEW

File: gtk2_ardour/po/cs.po [hunk 1]
```diff
@@ -17472,7 +17472,7 @@ msgid ""
 msgstr ""
 "Instalace ffmpeg nebyla v tomto systému nalezena.\n"
 "%1 vyžaduje ffmpeg a ffprobe z ffmpeg.org - verze 1.1 nebo novější.\n"
-"Zavádění a vyvádění obrazu není možné, dokud nenainstalujete nástroje.\n"
+"Zavádění a exportního obrazu není možné, dokud nenainstalujete nástroje.\n"
```
Comment (groundf) [inline gtk2_ardour/po/cs.po:17475]: Replace by "exportování".

File: gtk2_ardour/po/cs.po [hunk 2]
```diff
@@ -7723,7 +7723,7 @@ msgstr "Nahrává se '%3' (časový úsek %1 z %2)"
 
 #: export_dialog.cc:481
 msgid "Running Post Export Command for '%1'"
-msgstr "Provádí se povyváděcí příkaz pro '%1'"
+msgstr "Provádí se exportní příkaz pro '%1'"
```
Comment (kejml) [inline gtk2_ardour/po/cs.po:7726]:
```suggestion
msgstr "Provádí se post-exportní příkaz pro '%1'"
```

File: gtk2_ardour/po/cs.po [hunk 3]
```diff
@@ -7854,11 +7854,11 @@ msgstr "Vybrat složku pro vyvedení"
 
 #: export_format_dialog.cc:39
 msgid "New Export Format Profile"
-msgstr "Nový vyváděcí profil formátu"
+msgstr "nový profil exportního formátu"
```
Comment (kejml) [inline gtk2_ardour/po/cs.po:7857]:
```suggestion
msgstr "Nový profil exportního formátu"
```

File: gtk2_ardour/po/cs.po (hunk 2 above)
Comment (groundf) [reply gtk2_ardour/po/cs.po:7726]: What about "Provádí se post-export příkaz pro '%1'"?

**Comment by x42** [comment]
Is this ready to be merged, or are you still reviewing and working on it?

**Comment by groundf** [comment]
Hi, we are still reviewing this PR. This PR started a discussion on the local (Czech) forum, so I am waiting for a response. Is it OK to keep it open?  I will inform you when it is read (max. two weeks).

**Comment by x42** [comment]
Yes it's fine to keep this open. Also no need to rush. Ardour 7 is still a while away, and before release there will be a "call for translations" (on the forum, IRC and email-list).

**Comment by pauldavisthefirst** [comment]
We're now getting much, much closer to a v7.0 release, and so translations start to become important and on the critical path again.

**Comment by prokoudine** [comment]
@groundf Please let us know when we can merge the translation update

**Comment by luzpaz** [comment]
@groundf bump

**Comment by prokoudine** [comment]
@groundf Hi, could you please provide some insight to removing your work? Seems like it was a major effort to just let it go.

**Pull request closed without merging**